### PR TITLE
Changed from indexof to includes and var -> let #4

### PR DESCRIPTION
**Pull request by erinol**
**Testing Plan:** 

**GIFs or Screenshots:**

### CODE REVIEW

**Review by jacobsorme — APPROVED** [review]
:+1:

File: frontend_tests/node_tests/search_suggestion_legacy.js [hunk 1]
```diff
@@ -659,15 +659,15 @@ run_test('sent_by_me_suggestions', () => {
         return;
     };
 
-    var query = '';
-    var suggestions = search.get_suggestions_legacy(query);
-    assert(suggestions.strings.indexOf('sender:bob@zulip.com') !== -1);
+    let query = '';
+    let suggestions = search.get_suggestions_legacy(query);
+    assert(suggestions.strings.includes('sender:bob@zulip.com') !== -1);
```
Comment (mkforsb) [inline frontend_tests/node_tests/search_suggestion_legacy.js]: I don't think the `!== -1` is supposed to be here

**Review by mkforsb — CHANGES_REQUESTED** [review]

**Review by mkforsb — APPROVED** [review]
:+1:

**Comment by erinol** [comment]
Closes #4

**Comment by erinol** [comment]
Closes #4

**Pull request merged**